.gitmodules

```
@@ -98,3 +98,7 @@
 [submodule "projects/rocprofiler-systems/external/filesystem"]
 	path = projects/rocprofiler-systems/external/filesystem
 	url = https://github.com/gulrak/filesystem.git
+[submodule "projects/rocprofiler-systems/external/spdlog"]
+	path = projects/rocprofiler-systems/external/spdlog
+	url = https://github.com/gabime/spdlog.git
+	branch = v1.x
```

Use concrete version, maybe we can start with the latest release version [v1.16.0](https://github.com/gabime/spdlog/tree/v1.16.0)
```suggestion
	branch = v1.16.0
```